tests/cypress/integration/actions_tasks2/case_101_opencv_basic_actions.js

```diff
@@ -0,0 +1,128 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+import { taskName, labelName } from '../../support/const';
+
+context('OpenCV. Intelligent cissors. Histogram Equalization.', () => {
+    const caseId = '101';
+    const createOpencvShape = {
+        labelName: labelName,
+        pointsMap: [
+            { x: 200, y: 200 },
+            { x: 250, y: 200 },
+            { x: 300, y: 250 },
+            { x: 350, y: 300 },
+            { x: 300, y: 350 },
+        ],
+    };
+    const newLabel = `${createOpencvShape.labelName} new`
+    const createOpencvShapeSecondLabel = {
+        labelName: newLabel,
+        pointsMap: [
+            { x: 300, y: 200 },
+            { x: 350, y: 200 },
+            { x: 400, y: 250 },
+            { x: 450, y: 300 },
+            { x: 400, y: 350 },
+        ],
+        useDoneButton: true,
+    };
+    const keyCodeN = 78;
+    const pointsMap = [
+        { x: 300, y: 400 },
+        { x: 350, y: 500 },
+        { x: 400, y: 450 },
+        { x: 450, y: 500 },
+        { x: 400, y: 550 },
+    ];
+
+    function generateString(countPointsToMove) {
+        let action = '';
+        for (let i = 0; i < countPointsToMove; i++) {
+            action += '{rightarrow}';
+        }
+        return action;
+    }
+
+    function openOpencvControlPopover() {
+        cy.get('body').focus();
+        cy.get('.cvat-tools-control').trigger('mouseleave').trigger('mouseout').trigger('mouseover');
+    }
+
+    before(() => {
+        cy.openTask(taskName);
+        cy.addNewLabel(newLabel);
+        cy.openJob();
+    });
+
+    describe(`Testing case "${caseId}"`, () => {
+        it('Load OpenCV.', () => {
+            openOpencvControlPopover();
+            cy.get('.cvat-opencv-control-popover-visible').find('.cvat-opencv-initialization-button').click();
+            // Intelligent cissors button be visible
+            cy.get('.cvat-opencv-drawing-tool').should('exist').and('be.visible');
+        });
+
+        it('Create a shape with "Intelligent cissors".', () => {
+            cy.opencvCreateShape(createOpencvShape);
+        });
+
+        it('Create a shape with "Intelligent cissors" and "Done" button.', () => {
+            cy.opencvCreateShape(createOpencvShapeSecondLabel);
```

Comment:
Let's unite it into one test. In both cases, we test creation, just in different scenarios.

Reply:
Done.